### ISSUE TITLE
Exception when using it...

### ISSUE THREAD

**Issue by jogibear9988** [sashafirsov/html-demo-element#5]
I tried using your element in my designer:
https://node-projects.github.io/web-component-designer-demo/index.html

With this code:

        <html-demo-element legend="with legend attribute 🤗" description="👓
                        Description will be kept next to legend
                        but less emphasized ">Eating own dog food 🦴</html-demo-element>

I always get this exception:
<img width="755" alt="image" src="https://github.com/user-attachments/assets/b11fa7ed-564b-48cd-9782-6db89e251b9c">
<img width="555" alt="image" src="https://github.com/user-attachments/assets/eda3fd22-de24-4327-b754-4a392f5adba9">

<img width="1002" alt="image" src="https://github.com/user-attachments/assets/53032094-66ac-434a-b2f1-6de72fc3d8cc">

**Comment by sashafirsov** [comment]
@jogibear9988 , the sample in raw works as expected. Within your designer I was not able to replicate it either. 
Please upgrade to latest 1.0.9 where I added the safety checks for programmatic use. 
It should fix the exception but perhaps not your use case.

Please let me know if upgrade works.

**Comment by sashafirsov** [comment]
```html
<script type="module" src="https://unpkg.com/html-demo-element@1.0.10/html-demo-element.js"></script>
<html-demo-element legend="with legend attribute 🤗" description="👓
                    Description will be kept next to legend
                    but less emphasized ">Eating own<code>dog food</code>🦴</html-demo-element>
```
in code editor does the work in designer:
https://node-projects.github.io/web-component-designer-demo/index.html

![image](https://github.com/user-attachments/assets/ea7ac1d0-6f49-468a-86f7-b19936370610)
Only trouble is styling by **prysmjs** is broken :(

**Comment by jogibear9988** [comment]
I load it as a NPM package:

<img width="285" alt="image" src="https://github.com/user-attachments/assets/95002a3b-3ae6-4efc-b26d-ca38dd5bbd1c">

but now it seems to work

**Comment by sashafirsov** [comment]
the DOM part is OK, but prysm css is not. Network panel shows it is loaded but rules are not applied

**Comment by jogibear9988** [comment]
my designe ruses shadow dom, so your globals style injection will not work on nested webcomponents

**Comment by sashafirsov** [comment]
Styles would be addressed in #6 
closing
